servatrice/migrations/servatrice_0009_to_0010.sql

```diff
@@ -0,0 +1,12 @@
+-- Servatrice db migration from version 9 to version 10
+
+CREATE TABLE IF NOT EXISTS `cockatrice_warnings` (
+  `user_name` varchar(255) NOT NULL,
```

Comment:
I see your point as users can change their usernames overtime and I do agree we should probably store user ID > username